src/main/java/org/terasology/moduletestingenvironment/MTEExtension.java

```diff
@@ -65,6 +66,8 @@ public void afterAll(ExtensionContext context) throws Exception {
 
     @Override
     public void beforeAll(ExtensionContext context) throws Exception {
+        System.setProperty(ModuleManager.LOAD_CLASSPATH_MODULES_PROPERTY, "true");
```

Comment:
This is super sloppy: setting a global property in `beforeAll` and then never restoring it to its previous state.

but I'm tired and I rationalized it by telling myself that I don't think there are other test cases that will use a ModuleManager in the same process as an MTE test.

if there's a handy JUnit way to manage this, let me know.